src/sugar3/graphics/toggletoolbutton.py

```diff
@@ -58,6 +58,25 @@ def setup_accelerator(tool_button):
 
 
 class ToggleToolButton(Gtk.ToggleToolButton):
+    '''
+    UI for toggletoolbutton.
+    A ToggleToolButton is a ToolItem that contains a toggle button,
+    having an icon, a tooltip palette, and an accelerator.
+    Use ToggleToolButton.new() to create a new ToggleToolButton.
+
+    Fuctions:
+        ToggleToolButton()
+            Returns: a new toggletoolbutton
+
+    Args:
+        accelerator (string): keyboard shortcut to be used to
```

Comment:
What format should this be in?  I think that there is a documentation for the format on the GDK docs somewhere - maybe add a hyperlink to aid the reader.